frontend/src/TrackFile/MediaInfo.js

```diff
@@ -27,6 +28,16 @@ function MediaInfo(props) {
           !!audioChannels &&
           audioChannels.toFixed(1)
         }
+
+        {
+          !!audioCodec && !!audioChannels && !!audioBitRate &&
```

Comment:
Probably just me not being familiar with js, but is this what you meant to have here?

Reply:
Yea, just shorthand null checks and conditionals

Reply:
Could you express this with the shorthand as follows?  I didn't run a test to see if it works this way in JS 
```javascript
(!!(audioCodec && audioChannels && audioBitrate))
```
I think I've used this syntax in some ES6 and TypeScript, not sure if core ECMA supports it

Reply:
Looking at this closer, I think we really want:
`((!!audioCodec && !!audioBitRate) || (!!audioChannels && !!audioBitRate)) &&`

We want to render the separator if audioBitRate and one of the other two properties exists, not just when all 3 exist.